docs/config.md

```diff
@@ -0,0 +1,125 @@
+# Bifrost Configuration
+
+## ServiceSettings
+
+ServiceSettings is the configuration related to the web server.
+
+### Host
+
+*string*
+
+The URL that a Mattermost instance will use as S3 endpoint. The port number is required.
+
+### HealthHost
+
+*string*
+
+The URL that will be used by K8s so that it can ping this endpoint to know the state of the server.
+
+### TLSCertFile
+
+*string*
+
+The path to the certificate file to use for TLS connection security.
+
+### TLSKeyFile
+
+*string*
+
+The path to the TLS key file to use for TLS connection security.
+
+### MaxConnsPerHost
+
+*int*
+
+Maximum allowed number of connections per host.
+
+### ResponseHeaderTimeout
+
+*int*
+
+Specifies the amount of time to wait for a server's response headers after fully writing the request.
+
+## S3Settings
+
+Settings related to S3-compatible object storage instance. E.g. Amazon S3, Minio etc.
```

Comment:
I'd just avoid Minio for now. That's why the config struct is called `S3Settings`. Bifrost is just focused on S3 for now. It's good that minio is compatible with S3, but we don't guarantee that.